solutions/cses-2428.mdx

```diff
@@ -74,4 +74,39 @@ public class cses2428 {
 
 </JavaSection>
 
-</LanguageSection>
+<PySection>
+
+```py 
+n,k = map(int,input().split())
```

Comment:
space out operators

Reply:
n, k = map( int, input().split() )

Is this how you want the spaces if not please tell me.

Reply:
uh like
```suggestion
n, k = map(int, input().split())
```

Reply:
updated